schema-derivatives-perpfutures.graphql

```diff
@@ -1,5 +1,5 @@
 # Subgraph Schema: Derivatives Perpetual Futures
-# Version: 1.2.1
+# Version: 1.2.2
```

Comment:
This technically has breaking changes in the schema. By Semver, we would need to upgrade to 2.0.0, but since we have not created the pipeline for this in our BE yet, I think `1.3.0` is safe